plugins/woocommerce-admin/client/marketplace/contexts/subscriptions-context.tsx

```diff
@@ -55,6 +55,21 @@ export function SubscriptionsContextProvider( props: {
 	};
 
 	useEffect( () => {
+		/**
+		 * Check if we have &install=PRODUCT_KEY in the URL. This means we have just
+		 * installed a new product and nwe need to refresh the list.
+		 */
+		const urlParams = new URLSearchParams( window.location.search );
+		const installKey = urlParams.get( 'install' );
+
+		if ( installKey ) {
+			refreshSubscriptions().finally( () => {
```

Comment:
I would just the finally to `refreshSubscriptions` to make it consistent with `loadSubscriptions`.

Reply:
> I would just the finally to refreshSubscriptions to make it consistent with loadSubscriptions.

I didn't fully get this. I left it as it is, `refreshSubscriptions` can't handle the loading state unlike `loadSubscriptions` so it needs finally to do `setIsLoading( false );`.

Reply:
I see, I skipped some words.

I was thing of moving the `.finally( () => {
				setIsLoading( false );
			} );` code to `refreshSubscriptions`.
This way it would match `loadSubscriptions`.

Reply:
OK, got it. I'll try this now

Reply:
OK, done. I think I made `refreshSubscriptions` and `loadSubscriptions` consistent! Hope it looks good :crossed_fingers: